### DISCUSSION TITLE
install team talk on linux as a service

### DISCUSSION THREAD

**Discussion by farshad-fallah** [BearWare/TeamTalk5#2012] [General]
how i can install teamtalk on linux as a service  i mean teamtalk run automaticly after reboot on linux. i search in internet but i found just a web site volta.com  and says about init script.. i could not do that.is there has  any dowctument or some one help me.. thanks befor

**Comment by bear101** [comment]
You will need to edit the TeamTalk Server script your Linux distribution requires in order to set up the TeamTalk Server correctly. I.e. assign user account and paths to executable (tt5srv) and configuration (tt5srv.xml).
